addons/l10n_pe/data/l10n_latam_identification_type_data.xml

```diff
@@ -1,15 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <record model='l10n_latam.identification.type' id='l10n_latam_base.it_vat'>
-        <field name='l10n_pe_vat_code'>6</field>
```

Comment:
Maybe we can still add the code here?  Or does 6 mean that it is a vat from Peru?

Reply:
It means that can be used as a RUC, that's all.

Reply:
@umiphos So, the VAT can be used as RUC, so we can still add this code to it?

Reply:
yes, we can add the code to the VAT without a problem

Reply:
I think it is safer to do it then.

Reply:
@jco-odoo do you want me to leave the code for the VAT?

Reply:
Yes.  I think it is better (unless you would say it is not)

Reply:
done and ready

Reply:
@jco-odoo if you think this is ok, can you approve it?